addons/l10n_mx/__init__.py

```diff
@@ -2,3 +2,14 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from . import models
+
+from odoo import api, SUPERUSER_ID
+
+
+def add_debit_tag(cr, registry):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    debit_accounts = env['account.account'].search([('code', '=like', '102%')])
+    debit_tag = env['account.account.tag'].search([('name', '=like', 'Debit Balance Account')])
```

Comment:
maybe we could ref by `tag_debit_balance_account` id instead, so it is more robust? :slightly_smiling_face: